ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -349,6 +390,33 @@ private[apiserver] final class ApiUserManagementService(
         Future.successful(t)
     }
 
+  private def verifyPartyExistInIdp(
+      rights: Set[UserRight],
```

Comment:
do we need to include `primaryParty` here? from the docs `primaryParty` is not used directly and still has to be granted